### PR TITLE
AIC-75 Bugfixes

### PR DESCRIPTION
**Pull request by raigex**

### CODE REVIEW

File: search/src/main/java/edu/artic/search/SearchAudioDetailViewModel.kt [hunk 1]
```diff
@@ -58,7 +67,7 @@ class SearchAudioDetailViewModel @Inject constructor(private val analyticsTracke
 
         articObjectObservable
                 .map { it.location.orEmpty() }
-                .map { it.isEmpty() }
+                .map { !it.isEmpty() }
```
Comment (Cliabhach) [inline search/src/main/java/edu/artic/search/SearchAudioDetailViewModel.kt]: We could use `it.isNotEmpty()` too, right?

File: search/src/main/res/layout/fragment_search_audio_detail.xml [hunk 2]
```diff
@@ -54,6 +54,7 @@
                     android:contentDescription="@null"
                     android:fitsSystemWindows="true"
                     android:src="@color/audioBackground"
+                    android:adjustViewBounds="false"
```
Comment (Cliabhach) [inline search/src/main/res/layout/fragment_search_audio_detail.xml]: This was the default state, I think.

File: search/src/main/java/edu/artic/search/SearchAudioDetailViewModel.kt [hunk 3]
```diff
@@ -76,6 +85,13 @@ class SearchAudioDetailViewModel @Inject constructor(private val analyticsTracke
                 .bindTo(playAudioVisible)
                 .disposedBy(disposeBag)
 
+        articObjectObservable
+                .filterFlatMap({it.gallery != null} , {it.gallery!!})
+                .filter { it.number != null }
+                .map { "Gallery ${it.number.toString()}" } // TODO: use localizer
```
Comment (Cliabhach) [inline search/src/main/java/edu/artic/search/SearchAudioDetailViewModel.kt]: We can at least use a string resource here.

File: search/src/main/java/edu/artic/search/SearchAudioDetailViewModel.kt (hunk 1 above)
Comment (raigex) [reply search/src/main/java/edu/artic/search/SearchAudioDetailViewModel.kt]: true. updated

File: search/src/main/res/layout/fragment_search_audio_detail.xml (hunk 2 above)
Comment (raigex) [reply search/src/main/res/layout/fragment_search_audio_detail.xml]: most likely

File: search/src/main/java/edu/artic/search/SearchAudioDetailViewModel.kt (hunk 3 above)
Comment (raigex) [reply search/src/main/java/edu/artic/search/SearchAudioDetailViewModel.kt]: updated to pull generation of this full field into Fragment under protest

**Pull request merged**